### PR TITLE
Enable Python 3.10 support

### PR DESCRIPTION
**Pull request by fnordahl**
Please review commit by commit for details.

### CODE REVIEW

File: .github/workflows/tox.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
```
Comment (ajkavanagh) [inline .github/workflows/tox.yaml:12]: Looks like two spaces has crept in here?  Is this a new standard?

Comment (fnordahl) [reply .github/workflows/tox.yaml:12]: Thank you for the review @ajkavanagh , much appreciated!

I don't see the extra space though? I did convert all the integers to string versions for consistency though, as a bare `3.10` will be interpreted as `3.1`.

I wonder if GitHub's representation of that change poses as an optical illusion wrt. the number of spaces? :-)

Comment (ajkavanagh) [reply .github/workflows/tox.yaml:12]: > I wonder if GitHub's representation of that change poses as an optical illusion wrt. the number of spaces? :-)

I think that must be it!  I even pulled down the PR and checked.  How strange.  it must be the `', '` combination, highlighted in green, vs red for the previous version, that makes it look wider.

**Review by ajkavanagh — APPROVED** [review]
LGTM.

**Pull request merged**